ethereum/api/src/test/java/org/hyperledger/besu/ethereum/api/jsonrpc/internal/privacy/methods/priv/PrivDeletePrivacyGroupTest.java

```diff
@@ -0,0 +1,43 @@
+package org.hyperledger.besu.ethereum.api.jsonrpc.internal.privacy.methods.priv;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+import org.hyperledger.besu.enclave.Enclave;
+import org.hyperledger.besu.ethereum.api.jsonrpc.internal.JsonRpcRequest;
+import org.hyperledger.besu.ethereum.api.jsonrpc.internal.JsonRpcRequestContext;
+import org.hyperledger.besu.ethereum.api.jsonrpc.internal.response.JsonRpcError;
+import org.hyperledger.besu.ethereum.api.jsonrpc.internal.response.JsonRpcErrorResponse;
+import org.hyperledger.besu.ethereum.core.PrivacyParameters;
+
+import org.junit.Before;
+import org.junit.Test;
+
+public class PrivDeletePrivacyGroupTest {
```

Comment:
nit; you can use the Mockito Annotations (that'll avoid needing to declare the mocks with `mock(Enclave.class)` and would be more consistent with the majority of unit tests.

Reply:
A lot of the other test classes in this package aren't using annotations. In fact, only one is using the annotations as opposed to mock();

I think I'll keep this one as it is for now, given that I won't refactor every other test to use annotations. However, it would be good to converge the two approaches into one approach one day. :)

Reply:
Actually, I'll use the annotation in the two new test classes that I have created. I won't change the other test classes to use this pattern for now. 👍

Reply:
>I think I'll keep this one as it is for now, given that I won't refactor every other test to use annotations. However, it would be good to converge the two approaches into one approach one day. :)
😞